tutorials/deutsch_jozsa/deutsch_jozsa.ipynb

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
<p>please add the import in the next code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>cool</p>